src/libs/constants.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+//-------------------------------------------------------
+// IMPORTANT:
+// The HASH_ITERATIONS and HASH_KEY_SIZE constants below determine how the hashed auth passwords are generated and, most importantly, validated.
+// Changing any of these values will make all existing passwords on the database invalid and deny access to them to all the proposals users.
+//-------------------------------------------------------
+
+/**
+ * Number of iterations to execute when hashing the password
+ *
+ * NIST recommends at least 10,000 iterations (https://pages.nist.gov/800-63-3/sp800-63b.html#sec5),
+ * @see Comment above before changing this value
+ */
+export const HASH_ITERATIONS = 10000;
```

Comment:
We should benchmark this because it will run on a lambda with very limited resources, the lib [uses a lower value](https://github.com/HathorNetwork/hathor-wallet-lib/blob/25b81a822726f811979edd54a10ba5182b4f2db4/src/constants.js#L191) exactly because it was taking too long to hash the passwords

Not saying we should lower this, but test how long it will take on realistic scenarios

Reply:
Thanks for the heads up!

On local `serverless-offline` tests the average request response time rises from `40ms` to `280ms` when raising the hash iterations from 1000 to 10.000.

It seems to me that keeping the secret salt as well hashed as our wallets passwords is a good enough approach for this application, so I reverted it back to 1000.